Classes/Service/CacheService.php

```diff
@@ -71,6 +80,16 @@ public function flush($force = false)
     }
 
     /**
+     * Flushes caches using the data handler. This should not be necessary any more in the future.
+     * @deprecated
+     * @internal
+     */
+    public function flushCachesWithDataHandler()
+    {
+        $this->dataHandler->clear_cacheCmd('all');
```

Comment:
> We have a special customer extension, which trigger cache clearing actions to other systems

And the hook subscriber only does things when action "system" is called?

> But if this is deprecated, we need a new handling for that.

For now, the hook is still valid. But in TYPO3 8.x cache command "all" really means all caches, thus also system caches. If it is not too expensive, I suggest to flush caches on any command